azureDatabaseUtils/src/main/resources/application.yml

```diff
@@ -1,8 +1,14 @@
+azureEnvironmentConfig:
+  AzureUSGovernmentCloud: ${DB_SERVER_NAME}.postgres.database.usgovcloudapi.net
+  AzureCloud: ${DB_SERVER_NAME}.postgres.database.azure.com
```

Comment:
these are db host names, can you include that in the property name?

Reply:
Made subvalues of dbHostNames